internal/cmd/run.go

```diff
@@ -23,10 +23,10 @@ import (
 
 	"github.com/urfave/cli/v2"
 
+	"github.com/tetratelabs/getenvoy/internal/binary/envoy"
+	"github.com/tetratelabs/getenvoy/internal/binary/envoy/debug"
+	"github.com/tetratelabs/getenvoy/internal/globals"
 	internalreference "github.com/tetratelabs/getenvoy/internal/reference"
```

Comment:
now this sounds weird since we have all internal..
```suggestion
	"github.com/tetratelabs/getenvoy/internal/reference"
```

Reply:
also in internal/cmd/fetch.go

Reply:
agreed. I will hunt for these now